TileDB.Cloud/Udf.cs

```diff
@@ -0,0 +1,480 @@
+#nullable enable
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Polly;
+using TileDB.Cloud.Rest.Api;
+using TileDB.Cloud.Rest.Model;
+
+namespace TileDB.Cloud
+{
+    public partial class RestUtil
+    {
+        public static class Udf
```

Comment:
How about we name it `UserDefinedFunction`?

Reply:
The rename is appropriate [according to the Framework Design Guidelines](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/general-naming-conventions#using-abbreviations-and-acronyms).

> DO NOT use any acronyms that are not widely accepted, and even if they are, only when necessary.